File: Python/1061 - Tempo de um Evento.py
dia_inicio = int(input().split()[1])
h_inicio, m_inicio, s_inicio = map(int, input().split(" : "))

dia_fim = int(input().split()[1])
h_fim, m_fim, s_fim = map(int, input().split(" : "))

segundos_inicio = dia_inicio * 86400 + h_inicio * 3600 + m_inicio * 60 + s_inicio
segundos_fim = dia_fim * 86400 + h_fim * 3600 + m_fim * 60 + s_fim

duracao = segundos_fim - segundos_inicio

dias = duracao // 86400
duracao %= 86400

horas = duracao // 3600
duracao %= 3600

minutos = duracao // 60
segundos = duracao % 60

print(f"{dias} dia(s)")
print(f"{horas} hora(s)")
print(f"{minutos} minuto(s)")
print(f"{segundos} segundo(s)")

File: JavaScript/1061 - Tempo de um Evento.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n');
var diaInicio = parseInt(input[0].split(' ')[1]);
var [hInicio, mInicio, sInicio] = input[1].split(' : ').map(Number);
var diaFim = parseInt(input[2].split(' ')[1]);
var [hFim, mFim, sFim] = input[3].split(' : ').map(Number);

var inicio = sInicio + mInicio*60 + hInicio*3600 + diaInicio*86400;
var fim = sFim + mFim*60 + hFim*3600 + diaFim*86400;

var duracao = fim - inicio;

var dias = Math.floor(duracao / 86400); duracao %= 86400;
var horas = Math.floor(duracao / 3600); duracao %= 3600;
var minutos = Math.floor(duracao / 60);
var segundos = duracao % 60;

console.log(`${dias} dia(s)`);
console.log(`${horas} hora(s)`);
console.log(`${minutos} minuto(s)`);
console.log(`${segundos} segundo(s)`);